lms/templates/help_modal.html

```diff
@@ -15,7 +15,7 @@
 % if settings.FEATURES.get('ENABLE_FEEDBACK_SUBMISSION', False):
 
 <div class="help-tab">
-  <a href="#help-modal" rel="leanModal" role="button">${_("Help")}</a>
+  <a href="#help-modal" rel="leanModal" role="button">${_("Feedback")}</a>
```

Comment:
Yes and no. This one is is a little murky. We're using a page anchor (#anchor) rather than a fully qualified URL. When we follow links, we expect our current document location to change (title change, content change, URL change). If those things don't happen, we should use a button.

Because this link opens a new URL in a new browser window, we should just use a fully qualified URL. So, `<a href="http://edx.readthedocs.org/" ... ></a>`. The Lean Modal should pick it up, and if for some reason JavaScript is busted, the link will still work.